tests/unit/components/data-grid-test.js

```diff
@@ -15,6 +15,7 @@ test('with content: has a ArrayDataManager with content', function(assert) {
     dataGrid = this.subject({
       content: [1, 2, 3, 4]
     });
+    dataGrid.didReceiveAttrs();
```

Comment:
Why do you need this?

Reply:
Because this is no longer automatically done by the constructor in unit tests